apps/playground-api/src/PlaygroundBlock.ts

```diff
@@ -12,7 +12,9 @@ export class PlaygroundBlock {
 
   @Interval(3000)
   async generate (): Promise<void> {
-    await this.client.call('generatetoaddress', [1, PlaygroundSetup.address, 1], 'number')
+    const { masternodes }: MiningInfo = await this.client.mining.getMiningInfo()
+    const randomAddress = masternodes[Math.floor(Math.random() * masternodes.length)].operator
+    await this.client.call('generatetoaddress', [1, randomAddress, 1], 'number')
```

Comment:
You need to add tests to assert this is valid.

Reply:
added testcase